core/arch/arm/arm.mk

```diff
@@ -18,3 +18,10 @@ core-platform-cppflags += $(arm32-platform-cppflags)
 core-platform-cflags += $(arm32-platform-cflags)
 core-platform-aflags += $(arm32-platform-aflags)
 endif
+
+ifeq ($(CFG_WITH_PAGER),y)
+ifneq ($(CFG_DISABLE_CONCURRENT_EXEC),y)
```

Comment:
@pascal-brand-st-dev yes that would simplify things indeed. Let's examine the different cases to make sure we agree on the behavior. Suppose we have `$(call force,VAR,val,reason)`, then:
- If VAR is undefined, define it to 'val'. Do we need to display a message here? I don't think so.
- If VAR is defined and has value 'val', do nothing
- If VAR is defined and has a different value than 'val', force the new value and do $(warning  'Setting VAR to val [reason]') -- the current macro errors out, which is probably not a very good choice given the name 'force'.

Then the current chunk could be rewritten as:
```
ifeq ($(CFG_WITH_PAGER),y)
$(call force,CFG_DISABLE_CONCURRENT_EXEC,y,"required by CFG_WITH_PAGER")
endif
```

If that's OK I will create a PR.